extensions/exposition/source/Gateway.ts

```diff
@@ -68,6 +79,9 @@ export class Gateway extends Connector {
 
     const body = await request.parse()
 
+    if ('embed' in request && typeof body === 'object' && body !== null)
+      Object.assign(body, request.embed)
```

Comment:
NIT: the fact that for embed to work we need `body` to be an object technically allows us to stop request processing earlier if `Content-Type` if set to something not parseable as object. Do we have such logic as an interceptor or we do want to allow non object body with embeds define case when `embed` would be just silently ignored?

Reply:
`Content-types` that potentially may contain objects doesn't have to be them.

This is a valid JSON, which is not an object: `1`.

Reply:
Soo. What is our opinion on this if an endpoint defines `embeds` but receives non extendable payload? Current code will just silently ignore that, right?

Reply:
Yeah, I spend some time thinking about "proper" behavior in this case and didn't end up with anything better. I believe it is better, than throwing an exception that may be caused by a bad client request.

Any suggestions?

Reply:
Well. Let's keep as is until we gather sone real life usage experience on this.